app/renderer/components/urlBarIcon.js

```diff
@@ -11,6 +11,19 @@ const dndData = require('../../../js/dndData')
 const {isSourceAboutUrl} = require('../../../js/lib/appUrlUtil')
 const searchIconSize = 16
 
+const getIconCssClass = (ctx) => {
+  if (ctx.isSearch) {
```

Comment:
@gyandeeps I tested on the latest master and it looks good on about:prefenreces :-D